htdocs/install/mysql/migration/3.8.0-3.9.0.sql

```diff
@@ -604,3 +604,15 @@ INSERT INTO llx_c_tva(rowid,fk_pays,taux,recuperableonly,note,active) VALUES (14
 INSERT INTO llx_c_tva(rowid,fk_pays,taux,recuperableonly,note,active) VALUES (1482,  148, '7','0','VAT reduced rate',1);
 INSERT INTO llx_c_tva(rowid,fk_pays,taux,recuperableonly,note,active) VALUES (1483,  148, '5','0','VAT super-reduced rate', 1);
 INSERT INTO llx_c_tva(rowid,fk_pays,taux,recuperableonly,note,active) VALUES (1484,  148, '0','0','VAT Rate 0', 1);
+
+--Deal with holidays_user that do not have rowid
+CREATE TABLE llx_holiday_users_tmp
+(
+	rowid       integer NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Current key for this table is couple   fk_user / fk_type and current code should deals with this.
Why adding a rowid into this table ? I know it is a better practice but this is a freeze version and if there is no critical bug to excuse the change, we must discard it